src/app/onboard/member/page.tsx

```diff
@@ -0,0 +1,114 @@
+"use client";
+
+import React, { useState } from "react";
+
+import { Button } from "@/components/Button";
+
+const MembershipType = () => {
+  const [selectedMembership, setSelectedMembership] = useState<
+    "member" | "non-member" | null
+  >(null);
+
+  const handleButtonClick = (isMember: boolean) => {
+    setSelectedMembership(isMember ? "member" : "non-member");
+  };
+
+  const handleBackButtonClick = () => {
+    window.history.back();
+  };
+
+  const handleNextButtonClick = () => {
+    // Navigate to the home page
+    window.location.href = "/";
+  };
+
+  return (
+    <React.Fragment>
```

Comment:
```suggestion
    <>
```

This has the same behaviour and is more concise